src/test/java/uk/gov/pay/api/it/CreatePaymentITest.java

```diff
@@ -45,11 +45,10 @@
 
     @Test
     public void createCardPayment() {
-
         publicAuthMock.mapBearerTokenToAccountId(API_KEY, GATEWAY_ACCOUNT_ID, CARD);
 
         connectorMock.respondOk_whenCreateCharge(AMOUNT, GATEWAY_ACCOUNT_ID, CHARGE_ID, CHARGE_TOKEN_ID,
-                CREATED, RETURN_URL, DESCRIPTION, REFERENCE, null, PAYMENT_PROVIDER, CREATED_DATE, SupportedLanguage.ENGLISH, REFUND_SUMMARY,
+                CREATED, RETURN_URL, DESCRIPTION, REFERENCE, null, PAYMENT_PROVIDER, CREATED_DATE, SupportedLanguage.ENGLISH, Boolean.FALSE, REFUND_SUMMARY,
```

Comment:
(This applies in a lot of places)

I think it’s preferable to use `true` for test booleans to avoid the default for `boolean` fields as being `false`: this checks that the value is actually being passed all the way through